app/services/startups/level_up_eligibility_service.rb

```diff
@@ -59,7 +59,7 @@ def next_level
     end
 
     def next_level_unlock_date
-      @next_level_unlock_date ||= next_level.unlock_on
+      @next_level_unlock_date ||= next_level.unlock_at&.to_date
```

Comment:
The only place this `next_level_unlock_date` method is used is in the `eligibility` method in the same class, where it's simply used to check whether this value is in the future. I feel that this method can be deleted in favor of simply using the value in the origin method:

```ruby
def eligibility
  @eligibility ||= begin
    if next_level.blank?
      ELIGIBILITY_NOT_ELIGIBLE
    elsif next_level.unlock_at&.future? && regular_student?
      ELIGIBILITY_DATE_LOCKED
    # elsif ...
    end
end
```